src/main/java/duke/dukeexceptions/EmptyDate.java

```diff
@@ -0,0 +1,21 @@
+package duke.dukeexceptions;
+
+/**
+ * The exception for when no date is entered for commands that requires dates.
+ */
+public final class EmptyDate extends DukeExceptions {
```

Comment:
Should the name of this exception be EmptyDateException ? :thinking:

Reply:
Yeap! You are right! I'll make the changes!